types/node/globals.d.ts

```diff
@@ -311,6 +311,39 @@ declare class Buffer extends Uint8Array {
     values(): IterableIterator<number>;
 }
 
+//#region borrowed
+// from https://github.com/microsoft/TypeScript/blob/38da7c600c83e7b31193a62495239a0fe478cb67/lib/lib.webworker.d.ts#L633 until moved to separate lib
```

Comment:
Is there an issue filed for this on https://github.com/microsoft/TypeScript? I haven't found one and it would be useful to link to an issue as I was unaware this was a request.

Reply:
I've filed https://github.com/microsoft/TypeScript/issues/43692 to track this.

Reply:
Added note.